### ISSUE TITLE
MoBIE-io incompatible with bigdataviewer-core:10.2.2

### ISSUE THREAD

**Issue by K-Meech** [mobie/mobie-io#94]
The newest version of fiji ships with bigdataviewer-core:10.2.2 which is currently incompatible with mobie-io. Trying to build with this version errors with:
```java: org.embl.mobie.io.n5.loaders.N5ImageLoader.N5CacheArrayLoader is not abstract and does not override abstract method loadArray(long[],int[]) in bdv.img.cache.SimpleCacheArrayLoader```

This isn't currently an issue for MoBIE (as I think we have bigdataviewer-core:10.2.1 on the MoBIE update site?), but it causes issues in other plugins that only depend on mobie/mobie-io (like some of mine for the Schwab lab :)

@tischi  - do you think we could change mobie/mobie-io to the new bigdataviewer-core version? Also, was there a particular reason to put this on the MoBIE update sites, rather than relying on the one that comes with fiji?

**Comment by tischi** [comment]
> Do you think we could change mobie/mobie-io to the new bigdataviewer-core version?

Yes! We should definitely do this! @KateMoreva could you please look into the above error and update the bdv-core dependency in the pom? 

> Also, was there a particular reason to put this on the MoBIE update sites, rather than relying on the one that comes with fiji?

For some time we needed a more recent version than the one that shipped with Fiji.
We should then also update the bdv-core dependency in the pom of mobie-viewer-fiji.

Ideal would be to completely remove the version of bdv-core and rely on the scijava-parent-pom, but unf. this is not necessarily what is shipped with Fiji, because of the Java8 update site :-(

**Comment by K-Meech** [comment]
Ok - sounds good! I saw fiji seems to be using a new version of the pom-scijava too (32.0.0-beta-4) - so probably worth updating the pom-scijava version in both too. Although the 'beta' in the name is a bit weird.

**Comment by tischi** [comment]
> 32.0.0-beta-4

You could ask about that here: https://gitter.im/fiji/fiji
